ChainSafe/lodestar
# Pick only files with matching keystore file format in the keystore directory

**Motivation**

Currently, if a password.txt file is within the same folder as the keystores JSON within ./keystores, there is a fatal error that persists. This PR fixes this.

Closes #4116

**Steps to test or reproduce**

Have `password.txt` file in the keystore directory. Start the vc and confirm it does not crash.
